kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -54,10 +49,14 @@
       'channel-switcher': require('kolibri.coreVue.components.channelSwitcher'),
       'breadcrumbs': require('./breadcrumbs'),
       'section-nav': require('./section-nav'),
+      'search-page': require('./search-page'),
     },
     methods: {
-      toggleSearch() {
-        this.toggleSearch();
+      routeToSearchPage() {
+        this.$router.replace({
```

Comment:
when you click the search button it shouldn't be a router replace - that's an actual navigation. The router replace is just for updating the query string as you type